lib/src/components/NxVulnerabilityDetails/details/DetailLink.tsx

```diff
@@ -7,13 +7,19 @@
 /* eslint react/prop-types: 0 */
 import React, {FunctionComponent, ReactNode} from 'react';
 
+import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
+
+import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
+
 export interface Props {
   href: string;
   children: ReactNode | string;
 }
 
 const DetailLink: FunctionComponent<Props> = function DetailLink({href, children}) {
-  return <a href={href} target="_blank" rel="noreferrer">{children}</a>;
+  return <a href={href} target="_blank" rel="noreferrer">
+           {children} <NxFontAwesomeIcon icon={faExternalLinkAlt}/>
+         </a>;
```

Comment:
Minor: this is how we usually format this:
```suggestion
  return (
    <a href={href} target="_blank" rel="noreferrer">
      {children} <NxFontAwesomeIcon icon={faExternalLinkAlt}/>
    </a>
  );
```